.github/workflows/ci.snyk.yml

```
@@ -0,0 +1,18 @@
+name: Snyk
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check vulnerabilities
+        uses: snyk/actions/node@master
```

```suggestion
        uses: snyk/actions/node@0.3.0
```

We shouldn't use the `master` version of third-party actions